code/game/objects/items/weapons/cane.dm

```diff
@@ -53,4 +53,70 @@
 	else
 		SetName("cane shaft")
 		icon_state = "nullrod"
-		item_state = "foldcane"
\ No newline at end of file
+		item_state = "foldcane"
+
+/obj/item/weapon/cane/crutch
+	name ="crutch"
+	desc = "A long stick with a crosspiece at the top, used to help with walking."
```

Comment:
icon = 'icons/urist/items/uristweapons.dmi

Repath icons to this, as long as they keep the same icon_state/item state names, they should work fine